SupersonicSound/LowLevel/LowLevelSystem.cs

```diff
@@ -221,11 +221,19 @@ void IPreInitilizeLowLevelSystem.SetCallback(Action<LowLevelSystem, SystemCallba
         #endregion
 
         #region init/close
+        [Obsolete("Not sure why this method is in here, the system should already be initialized when passed in through the constructor")]
         public void Initialize(int maxchannels, LowLevelInitFlags flags)
         {
             _system.init(maxchannels, (INITFLAGS) flags, IntPtr.Zero).Check();
         }
 
+        [Obsolete("This should only be called when we've initialized a low level system without studio (I think)")]
```

Comment:
I updated this now so it disposes the system if it's created without the Studio. And I removed the init and close/release methods since it's now handled from the constructor, like Studio.System. See if that works.